src/mpi/coll/ineighbor_alltoallw/ineighbor_alltoallw.c

```diff
@@ -19,8 +19,9 @@
       scope       : MPI_T_SCOPE_ALL_EQ
       description : |-
         Variable to select ineighbor_alltoallw algorithm
-        auto   - Internal algorithm selection
-        linear - Force linear algorithm
+        auto            - Internal algorithm selection
+        linear          - Force linear algorithm
+        gentran_linear  - Force generic transport based linear algorithm
 
```

Comment:
Also need add case to `MPIR_Ineighbor_alltoallw_impl` to the switch of `MPIR_Ineighbor_alltoallw_intra_algo_choice`.